### PR TITLE
Complete HACDOCS-434 work

### PR DESCRIPTION
**Pull request by Chr1st1anSears**
This MR consolidates the SBOM CLI and web UI docs into one, it renames that doc, and it puts the new doc in a new folder, "Secure your supply chain."

### CODE REVIEW

File: docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc [hunk 1]
```diff
@@ -1,26 +1,56 @@
...
 
 :toc: left
 :icons: font
-:numbered:
 :source-highlighter: highlightjs
 
-== Software bill of materials
-A software bill of materials (SBOM) provides greater transparency for your software supply chain. In {ProductName}, an SBOM is a list of all the software libraries that a xref:../glossary/index.adoc#_component[component] uses. Those libraries might enable specific functionality, facilitate development, or serve some other purpose. 
+== Software Bill of Materials
+A software bill of materials (SBOM) provides greater transparency for your software supply chain. In {ProductName}, an SBOM lists all the software libraries that a xref:../glossary/index.adoc#_component[component] uses. Those libraries may enable specific functionality, facilitate development, or serve some other purpose.
```
Comment (jc-berger) [inline docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc:8]: Am I incorrect that it's `{Product_Name}`

Comment (jc-berger) [reply docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc:8]: `In {Product_Name}, **a** SBOM lists all the software libraries that a component uses.`

I don't think we need to hyperlink to the component doc. I think linking would honestly make users leave and never return to this page. 

If you'd like, we can include an `Additional resources` section at the end where we can link to the glossary.

Comment (jc-berger) [reply docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc:8]: IBM SG discourages the use of `may` 

What do you think of: 

`Those libraries can enable specific functionality or facilitate development.` 

I think `serve some other purpose` is a little vague, so I'd recommend removing it.

File: docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc [hunk 2]
```diff
@@ -1,26 +1,56 @@
...
+== Software Bill of Materials
+A software bill of materials (SBOM) provides greater transparency for your software supply chain. In {ProductName}, an SBOM lists all the software libraries that a xref:../glossary/index.adoc#_component[component] uses. Those libraries may enable specific functionality, facilitate development, or serve some other purpose. 
 
-You can use an SBOM to better understand the composition of your software, identify vulnerabilities, and assess the potential impact of any security issues that might arise. You might also need to provide your customers with an SBOM to comply with cybersecurity regulations. 
+You can use an SBOM to better understand your software’s composition, identify vulnerabilities, and assess the potential impact of any security issues that may arise. You may also need to provide your customers with a component’s SBOM to comply with cybersecurity regulations.
```
Comment (jc-berger) [inline docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc:10]: IBM SG discourages use of `may`. Although it's often wordier, they discourage us using possessive for technology, so not `software's` 

Also, am I missing something? Why are we using `an` for SBOM? SHouldn't it be `a`? 
What do you think of:

`Use a SBOM to better understand the composition of your software, identify vulnerabilities, and asses the potential impact of any security issues. To comply with cybersecurity regulations, you might need to provide your customer with a SBOM component.` 

Note, though it's not a hard "must" we often are encouraged to lead with the "why" (comply with cybersecurity) then get to the action (provide component).

File: docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc [hunk 3]
```diff
@@ -1,26 +1,56 @@
...
+= Inspecting SBOMs
 
 :toc: left
 :icons: font
-:numbered:
 :source-highlighter: highlightjs
 
-== Software bill of materials
-A software bill of materials (SBOM) provides greater transparency for your software supply chain. In {ProductName}, an SBOM is a list of all the software libraries that a xref:../glossary/index.adoc#_component[component] uses. Those libraries might enable specific functionality, facilitate development, or serve some other purpose. 
+== Software Bill of Materials
```
Comment (jc-berger) [inline docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc:7]: Why are `Bill` and `Materials` capitalized? They don't read as proper nouns, so I wouldn't think they ought to be capitalized. But perhaps I'm missing something?

File: docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc [hunk 4]
```diff
@@ -1,26 +1,56 @@
...
+
+* You must have an application that {ProductName} has successfully built and deployed.
+
+.Procedure
+//Replace all images here
+Complete the following steps to download a component's SBOM:
+
+. In the console, navigate first to the *Activity* tab, then to the *Pipeline runs* tab.
++
+image::sbom/activity-to-pipeline-runs.png[alt=Pipeline runs tab]
```
Comment (jc-berger) [inline docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc:24]: I'm not a fan of including screenshots in our documentation. I see the reasoning behind it, so I'm not totally against it. 

But I think screenshots can be difficult to maintain and difficult to ensure accessibility. We also provide a lot of in-app documentation, so I don't think we need to include screenshots in our doc set. 

In short, my vote is to remove them.

File: docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc [hunk 5]
```diff
@@ -1,26 +1,56 @@
...
+.Prerequisites
+
+* You must have an application that {ProductName} has successfully built and deployed.
+
+.Procedure
+//Replace all images here
+Complete the following steps to download a component's SBOM:
+
+. In the console, navigate first to the *Activity* tab, then to the *Pipeline runs* tab.
++
```
Comment (jc-berger) [inline docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc:23]: What do you think of this rewrite?: 

In the console, complete the following steps to download a SBOM component:

1. Navigate to the *Activity* > *Pipeline runs* tab.

File: docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc [hunk 6]
```diff
@@ -1,26 +1,56 @@
...
+
+. For the component whose SBOM you want to view, select its most recent pipeline run.
+
+. From the *Pipeline run details* page, select the *Logs* tab. 
++
+image::sbom/pipeline-runs-details-banner.png[alt=Banner with logs tab]
+
+. In the navigation bar, select *show-sbom*.
+.. You can use your web browser to immediately search the SBOM for terms that indicate vulnerabilities in your software supply chain. For example, try searching for "log4j".
+.. Above the SBOM logs, you can select *Download* to download the SBOM, or *Expand* to view it full-screen.
```
Comment (jc-berger) [inline docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc:34]: Typically, we're discouraged from using directional words like `above`. Though I don't think it needs to be removed here, I think we could remove it and the step reads just as fine. 

`You can select *Downloads* to download the SBOM or *Expand* to view it full-screen.`

**Review by jc-berger — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc (hunk 1 above)
Comment (Chr1st1anSears) [reply docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc:8]: It is {ProductName} :+1:  and thanks for the minimalist suggestions!

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**